framework/test/unit/application.spec.ts

```diff
@@ -668,6 +668,7 @@ describe('Application', () => {
 		let _nodeDBSpy: jest.SpyInstance<any, unknown[]>;
 
 		beforeEach(async () => {
+			config.rootPath = '/lisk';
```

Comment:
May be instead of changing current ones, we should add a new tests case particularly for absolute root path.